### PR TITLE
add confirmation pin logic using validate pin for registration

### PR DESCRIPTION
**Pull request by Archifer**

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -148,21 +153,63 @@
...
+                } else {
+                  navigator.notification.alert(message);
+                }
+
+                this.cancelRegistration();
+              });
+            }
+          };
+
+        window.plugins.pinDialog.prompt('Create your' + options.pinLength + ' digit pin', confirmCallback, 'Register', ['Create', 'Cancel']);
```
Comment (MichielVrins) [inline src/views/Login.vue:203]: use String template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals?retiredLocale=nl

File: src/views/Login.vue [hunk 2]
```diff
@@ -148,21 +153,63 @@
...
+      pinHandler: function (actions, options) {
+        let confirmCallback = (results) => {
+            if (results.buttonIndex === 1) {
+              let confirmPin = results.input1;
+              onegini.user.validatePinWithPolicy({
+                pin: confirmPin
+              })
+              .then(() => {
+                let callback = (results) => {
+                  if (results.buttonIndex === 1) {
```
Comment (MichielVrins) [inline src/views/Login.vue:165]: maybe a comment to indiciate what index is which button.

Comment (Archifer) [reply src/views/Login.vue:165]: the plugin documentation contains the explanation of which index refers to which button + there are also no comments on all the other places, so I think it is oke

File: src/views/Login.vue [hunk 3]
```diff
@@ -148,21 +153,63 @@
...
-            let callback = (results) => {
-              if (results.buttonIndex === 1) {
-                actions.createPin(results.input1);
-              } else {
-                actions.cancel();
-              }
-            };
-
-            window.plugins.pinDialog.prompt('Create your ' + options.pinLength + ' digit pin', callback, 'Register', ['Create', 'Cancel']);
+            this.pinHandler(actions, options)
```
[the remark this answers is not in the document]
Comment (Archifer) [reply src/views/Login.vue:212]: will do

**Review by MichielVrins — APPROVED** [review]

**Pull request merged**